pkg/cluster/diff/diff.go

```diff
@@ -233,6 +240,53 @@ func getFileLines(path string) ([]string, string, error) {
 			}
 		}
 
+		if shortDiff && keep {
+			trimmedLine := strings.TrimLeft(line, " ")
+
+			// Skip over k2 annotations chunk in metadata since it's constantly
+			// changing and causing users to miss important changes.
+			if strings.HasPrefix(trimmedLine, "annotations:") {
+				insideAnnotation = true
+				keep = false
+			} else if insideAnnotation {
+				log.Debug("leading spaces: ", strconv.Itoa(countLeadingSpaces(line)))
+				// harcode labels here instead of checking the spaces
+				// since annotation is too long and it will then show
+				// some of the diff in annotation
+				if strings.HasPrefix(trimmedLine, "labels:") {
+					log.Debug("outside annotations: ", line)
+					insideAnnotation = false
+				} else {
+
+					keep = false
+
+				}
+			}
+
+			// Skip over k2 labels chunk in metadata since it's constantly
+			// changing and causing users to miss important changes.
+			if strings.HasPrefix(trimmedLine, "labels") {
+				insideLabels = true
+				keep = true
+			} else if insideLabels {
+				log.Debug("leading spaces: ", strconv.Itoa(countLeadingSpaces(line)))
+				if countLeadingSpaces(line) == 2 || countLeadingSpaces(line) == 6 {
+					log.Debug("outside labels: ", line)
+					insideLabels = false
+				} else {
+					if strings.HasPrefix(trimmedLine, "k2.segment.com") || strings.HasPrefix(trimmedLine, "app.kubernetes.io/instance:") || strings.HasPrefix(trimmedLine, "version: ") {
```

Comment:
Fix line length here.

Reply:
Style nit- be consistent in the prefixes you're looking for, e.g. `k2.segment.com:`, `app.kubernetes.io/instance:`, `version:`, etc.

Reply:
Will do! FYI, k2.segment.com/*, but will update the other two to be consistent.

Reply:
Ahh, ok, got it. Thanks!